Sources/SpeziOnboarding/SpeziOnboarding.docc/ObtainingUserConsent.md

```diff
@@ -28,6 +28,7 @@ OnboardingConsentView(
     action: {
         // Action to perform once the user has given their consent
     },
+    identifier: "MyFirstConsentForm", // Specify an optional unique identifier for the consent form, helpful for distinguishing consent forms when storing.
```

Comment:
It would be great if we can also use the code example as a way to explain the usage of enums to avoid "magic strings" as identifiers as noted in the other code comment.

Reply:
I added an example to the document.